README.md

```diff
@@ -1,2 +1,30 @@
-# HW01
-Edit the readme
+# HW01: A Brief Introduction of Myself
+
+## Who am I
+
+Hi there! I am **Yujia Liu** and you can call me **Randolph**. I am from **China (mainland)**. I spent a fall semester and a summer at UC Berkeley. It was a great time and I made incredible friends. During that time, I made a [webpage](http://isolates.genomics.lbl.gov) to help them sort out the isolates of microbes. My academic interest cross a wide range, including microbiology, genomics and protein structures. But overall, I am interested in how to apply quantitative methods to study these questions. I really hope the BMB program here at UChicago will motivate me to explore more on these topics!
```

Comment:
I am very impressed by your website. It looks great and seems like it would be quite useful!